docs/release-notes.rst

```diff
@@ -74,10 +74,14 @@ Upcoming Version (Not Yet Released)
    in the "stable" docs (corresponding to the last release) but will be visible in the
    "latest" docs (corresponding to the master branch).
 
+0.10.1 (Mar 22, 2021)
+---------------------
+
 Bug Fixes
 .........
 
-- Fixes an issue where automatic versioning erroneously treated classes as
+- Fixed an issue with hashing of strings containing invalid unicode.
```

Comment:
If we changed the cache schema version, then shouldn't we have a section on "Cache Compatibility Changes"? Also, then this would need to be 0.11, right?

Reply:
Good catch. Bumping to 0.11.0 instead of 0.10.1. Closing this PR and opened https://github.com/square/bionic/pull/352